skimage/data/__init__.py

```diff
@@ -60,8 +157,43 @@ def load(f, as_gray=False):
         Image loaded from ``skimage.data_dir``.
     """
     use_plugin('pil')
-    return imread(_os.path.join(data_dir, f), as_gray=as_gray)
-
+    return imread(fetch(f), as_gray=as_gray)
+
+
+def quantitative_phase_cells():
+    """Image of two cells retrieved from a digital hologram.
```

Comment:
This docstring definitely needs work, but I wanted to get the framework down and find the issue with the framework, rather than format docstrings for something that may never happen.